app/policies/mvi_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+MviPolicy = Struct.new(:user, :mvi) do
+  def access?
```

Comment:
maybe this could be `query?` (or a better term for has the needed attributes to query) i.e. `access?` is not a magic pundit symbol you can choose whatever you like :)